src/System.Net.HttpListener/src/System/Net/HttpListenerPrefixCollection.cs

```diff
@@ -39,6 +39,11 @@ internal HttpListenerPrefixCollection(HttpListener listener)
         public void CopyTo(Array array, int offset)
         {
             _httpListener.CheckDisposed();
+            // Preserve compatability with full .NET Framework.
+            if (array == null)
+            {
+                throw new NullReferenceException();
```

Comment:
Sure, I've removed the check. I just wanted to make it explicit that we want to do this